package/MDAnalysis/transformations/fit.py

```diff
@@ -181,12 +181,12 @@ def fit_rot_trans(ag, reference, plane=None, weights=None):
     ref, mobile = align.get_matching_atoms(reference.atoms, ag.atoms)
     try:
```

Comment:
I would suggest that you now remove the whole `try` `except` block here. `align.get_weights` will raise nicely written error messages.

Reply:
Sir, tried removing the block, but error still remains as it is, Please help